pkg/microservice/aslan/core/common/service/workflow.go

```diff
@@ -89,7 +89,6 @@ func DeleteWorkflow(workflowName, requestID string, isDeletingProductTmpl bool,
 	err = ProcessWebhook(nil, workflow.HookCtl.Items, webhook.WorkflowPrefix+workflow.Name, log)
```

Comment:
Is it necessary to synchronize? Can it be changed to asynchronous?

Reply:
not necessary